src/instant/instantForm.ts

```diff
@@ -109,7 +108,16 @@ export enum ProgressKind {
   proxyAllowancePayWithERC20 = 'proxyAllowancePayWithERC20',
 }
 
+export interface TradeTxSummary {
+  gasUsed: BigNumber;
+  sold: BigNumber;
+  soldToken: string;
+  bought: BigNumber;
+  boughtToken: string;
+}
+
 export type Progress = {
+  summary?: TradeTxSummary
```

Comment:
I don't understand why trade summary attributes are not just optional attributes of Progress interface?

Reply:
Right now after I see your comment I realize that I thought that `Progress { done: boolean }` was different type so I wanted to have summary object  when the TX is done. Then I saw that this was just extending the existing objects but never refactored it. 

Will do according to this comment and the latter one ( merging both amounts to the progress itself )